projects/fsastorefrontlib/src/occ/occ-models/occ.models.ts

```diff
@@ -81,10 +81,17 @@ export interface OBOConsentList {
 
 export interface OBOCustomerList {
   entries: FSUser[];
-  pagination: PaginationModel;
+  pagination: FSPaginationModel;
   sorts: SortModel[];
 }
 
+export interface FSPaginationModel extends PaginationModel {
```

Comment:
This looks like the `Pagination` interface that is already available. We should perhaps discuss the use of this, because it's in the `unused.model` indicating that it perhaps shouldn't be used at all. If that is the case maybe we should adjust our model on BE as well?

Reply:
There are 2 pagination models on BE and one of them is deprecated which is unfortunately used by Spartacus. They still don't have the newest one. I would expect that they have some kind of converter/normalizer that will transform the deprecated BE model into an appropriate FE model. If it does not exist, we can keep FSPaginationModel on our side until they don't fix it.

Reply:
But i agree, in that case, no need to extend existing PaginationModel, just create new

Reply:
I could find only `CUSTOMER_SEARCH_PAGE_NORMALIZER` and I'm not sure if we could use it, since there is a difference in `entries` type, they use `User` and we `FSUser` (FSUser extends User and has a few properties more).

Reply:
As agreed with @bojanjevtic-spa I'll use `Pagination`.